### DISCUSSION TITLE
Allocation notes

### DISCUSSION THREAD

**Discussion by WadeRolfe** [stayonhire/feedback#27] [General]
Hi team, in the allocations tab on a job, how do we right a specific note into each line item? I have tried clicking on this section but nothing allows me to add in a note. Please advise an easy way that I'm obviously missing. Thanks.

![Capture](https://github.com/stayonhire/feedback/assets/141391069/d35b9541-3209-46d1-bfef-9628d4dc29e4)

**Comment by twrayden** [comment]
I will look at this today, thanks for the feedback!

**Comment by twrayden** [comment]
Hi Wade,

The app has been updated so you should be able to edit the notes from that grid now. If you double click the notes cell that you want to edit, you should see a box where you can type your notes in. Lastly, to save you can press enter on your keyboard or just click away.

If you find that you still can't edit, you may need to refresh the page to ensure you are on the latest version, but let us know here if you have any more trouble with this one.

![image](https://github.com/stayonhire/feedback/assets/28379549/0386961e-f189-470e-b8b2-98992d8f885d)
